README.md

```diff
@@ -34,53 +34,79 @@ Microsoft AD As Built Report is a PowerShell module which works in conjunction w
 Please refer to the AsBuiltReport [website](https://www.asbuiltreport.com) for more detailed information about this project.
 
 # :beginner: Getting Started
+
 Below are the instructions on how to install, configure and generate a Microsoft AD As Built report.
 
 ## :floppy_disk: Supported Versions
 <!-- ********** Update supported AD versions ********** -->
-The Microsoft AD As Built Report supports the following AD versions;
+The Microsoft AD As Built Report supports the following Active Directory versions;
+
+- 2008, 2008 R2, 2012, 2016, 2019
 
 ### PowerShell
+
 This report is compatible with the following PowerShell versions;
 
 <!-- ********** Update supported PowerShell versions ********** -->
 | Windows PowerShell 5.1 |     PowerShell 7    |
 |:----------------------:|:--------------------:|
 |   :white_check_mark:   | :white_check_mark: |
+
 ## :wrench: System Requirements
 <!-- ********** Update system requirements ********** -->
 PowerShell 5.1 or PowerShell 7, and the following PowerShell modules are required for generating a Microsoft AD As Built report.
 
 - [AsBuiltReport.Microsoft.AD Module](https://www.powershellgallery.com/packages/AsBuiltReport.Microsoft.AD/)
+- [ActiveDirectory Module](https://docs.microsoft.com/en-us/powershell/module/activedirectory/?view=windowsserver2019-ps)
+- [PSPKI Module](https://www.powershellgallery.com/packages/PSPKI/3.7.2)
+- [GroupPolicy Module](https://docs.microsoft.com/en-us/powershell/module/grouppolicy/?view=windowsserver2019-ps)
 
 ### Linux & macOS
-* .NET Core is required for cover page image support on Linux and macOS operating systems.
-    * [Installing .NET Core for macOS](https://docs.microsoft.com/en-us/dotnet/core/install/macos)
-    * [Installing .NET Core for Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux)
+
+- .NET Core is required for cover page image support on Linux and macOS operating systems.
+  - [Installing .NET Core for macOS](https://docs.microsoft.com/en-us/dotnet/core/install/macos)
+  - [Installing .NET Core for Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux)
 
 ❗ If you are unable to install .NET Core, you must set `ShowCoverPageImage` to `False` in the report JSON configuration file.
```

Comment:
May require an additional note to highlight that this will not currently work on Linus & MacOS

Reply:
!Done